crates/index/benches/ray.rs

```diff
@@ -87,8 +65,8 @@ fn setup_world(world: &mut World, num_entities: u32, max_distance: f32) {
     }
 
     let mut rays = Rays::new();
-    let ray_origins = load_points(1000);
-    let ray_dirs = load_points(1000);
+    let ray_origins = load_points(&1000.try_into().unwrap(), MAP_SIZE);
```

Comment:
should we just use the enum variant here?

Reply:
I would.